stratum/glue/init_google.h

```diff
@@ -5,21 +5,10 @@
 #ifndef STRATUM_GLUE_INIT_GOOGLE_H_
 #define STRATUM_GLUE_INIT_GOOGLE_H_
 
-// TODO(boc): google only: this is not required for open source
-// #include "base/init_google.h"  // IWYU pragma: export
-
-// InitGoogle is not defined in portable base, so we instead emulate its
-// behavior.
-#if !GOOGLE_BASE_HAS_INITGOOGLE
-
 #include "gflags/gflags.h"
 #include "stratum/glue/logging.h"
 #include "stratum/glue/stamping.h"
 
-// TODO(unknown) need to define transformation or comment this out on
-// Google's side
-using gflags::ParseCommandLineFlags;
-
 inline void InitGoogle(const char* usage, int* argc, char*** argv,
```

Comment:
Rename to `InitLogging`

Reply:
On second thoughts, this function is really about gflags, not logging. And renaming it would make the filename incorrect as well. Let's postpone this for later.